test.py

```diff
@@ -0,0 +1,58 @@
+import torch
```

Comment:
Please move all the tests to a proper directory inside the parent `test` folder.